src/runtime/virtcontainers/api.go

```diff
@@ -195,16 +195,6 @@ func FetchSandbox(ctx context.Context, sandboxID string) (VCSandbox, error) {
 		return nil, err
 	}
 
-	// If the agent is long live connection, it needs to restart the proxy to
-	// watch the guest console if it hadn't been watched.
-	if s.agent.longLiveConn() {
-		err = s.startProxy()
```

Comment:
May this lose guest console output ?

Reply:
yes, it will be. previously, when we use vsock as connection socket, different hypervisors use different methods to deliver guest console output, that is, qemu uses shim, firecracker uses another goroutine, etc.